### PR TITLE
Add response handler support to ParallelQuerySet

### PR DESCRIPTION
**Pull request by drewaogle**
Move response_handler generation into ParallelQuery, simplyfing execute_batch.
Adds response_handler for ParallelQuerySet, defined as set_number,query,query_blobs,response,response_blobs.

### CODE REVIEW

File: aperturedb/ParallelQuery.py [hunk 1]
```diff
@@ -140,6 +110,44 @@ def setSuccessStatus(cls, statuses: list[int]):
...
+            end = start + commands_per_query
+            blobs_start = i * blobs_per_query
+            blobs_end = blobs_start + blobs_per_query
+
+            b_count = 0
+            if issubclass(type(response), list):
+                for req, resp in zip(query[start:end], response[start:end]):
+                    for k in req:
+                        # Ref to https://docs.aperturedata.io/query_language/Reference/shared_command_parameters/blobs
+                        blobs_where_default_true = \
```
Comment (gsaluja9) [inline aperturedb/ParallelQuery.py:130]: This is not correct anymore. blobs=False is default behavior

**Review by gsaluja9 — CHANGES_REQUESTED** [review]
response blob counted needs to be changed to align with adb blobs=false for find*

File: aperturedb/ParallelQuery.py (hunk 1 above)
Comment (drewaogle) [reply aperturedb/ParallelQuery.py]: fixed.

File: test/test_ResponseHandler.py [hunk 2]
```diff
@@ -220,3 +304,46 @@ def test_response_blobs(self, db, utils, monkeypatch):
...
+        self.responses = {}
+        self.response_blobs = {}
+        generator = QGSetPersonAndImages(self.requests, self.responses,
+                                         max_commands, self.response_blobs)
+        querier = ParallelQuerySet(db)
+        querier.query(generator, batchsize=99,
+                      numthreads=31,
+                      stats=True)
+
+        def expected_blobs_for_resp(rh_responses):
```
Comment (gsaluja9) [inline test/test_ResponseHandler.py:323]: this can be removed as it is not being used

**Review by gsaluja9 — DISMISSED** [review]

**Review by gsaluja9 — APPROVED** [review]

**Pull request merged**